cmd/main.go

```diff
@@ -80,7 +80,8 @@ const (
 )
 
 func main() {
-	logger, err := logger.New(os.Stdout, defLogLevel)
+	logLvl := mainflux.Env(envLogLevel, defLogLevel)
```

Comment:
I would be better if you can do first `loadConfig()` and then use `cfg.logLevel`. For that you could create a function to load agent conf and where you need the logger.

Reply:
I need logger before I can get configuration from the bootstrap service when I'm actually getting the conf file for agent , so I'll create logger with env and then after succsefull bootstrap ill recreate logger

Reply:
What I meant is that you should have in the main something like:
```
cfg := config {
	logLevel:   mainflux.Env(envLogLevel, defLogLevel),
	natsURL:   mainflux.Env(envNatsURL, defNatsURL),
	httpPort:   mainflux.Env(envHTTPPort, defHTTPPort),
	configFile: mainflux.Env(envConfigFile, defConfigFile),
}
```
This should be set with loadConfig().

You should then instantiate the logger and pass `cfg` and `logger` to `loadBootstrapConfig(cfg, logger)`